spec/factories.rb

```diff
@@ -119,14 +119,14 @@
     transient do
       title "An Unpublished Point"
       body "Some Body Text"
-      summary "A summary"
+      description "A summary"
```

Comment:
Worth making this transient in `unpublished_guide`, and then this can just inherit from `unpublished_guide` and just override the class? Guides have descriptions too right?